test/autotest.py

```diff
@@ -506,6 +506,8 @@ def testKill():
   def testCheckpoint():
     #start checkpoint
     coordinatorCmd(CKPT_CMD)
+    #wait and see if process will die shortly after checkpoint
+    sleep(S*SLOW)
```

Comment:
I am not sure if this 'sleep' really serves the purpose. It's quite possible that this sleep will overlap with the time it takes to perform ckpt. I think you intended to 'sleep' to be _after_ the ckpt image has been written. In which case, this sleep statement should go after the ckpt image verification has taken place. In fact, this should come after between calls to `testCheckpoint()` and `testKill()`. To be precise, the code should look like the following:
```
testCheckpoint()
sleep(S*SLOW)
CHECK(doesStatusSatisfy(getStatus(), status),
printFixed("PASSED ")
testKill()
```